flink-runtime/src/main/java/org/apache/flink/runtime/rest/RestClient.java

```diff
@@ -77,14 +77,10 @@
 
 	private static final ObjectMapper objectMapper = RestMapperUtils.getStrictObjectMapper();
 
-	// used to open connections to a rest server endpoint
-	private final Executor executor;
-
 	private Bootstrap bootstrap;
 
-	public RestClient(RestClientConfiguration configuration, Executor executor) {
+	public RestClient(RestClientConfiguration configuration) {
```

Comment:
Given that now we let netty do all the parsing, I think it makes sense to allow the user to specify the size of the `NioEventLoopGroup ` (currently set to 1 - which can be the default).